src/components/description/index.js

```diff
@@ -0,0 +1,13 @@
+import classes from './styles.module.css';
+import { descriptionValue } from '../utils/constants';
+
+const Description =() => {
+    return <div className={classes.descriptionContainer}>
```

Comment:
why do we need a separate container class, content class can handle all the styles of container since there is only a single element

Reply:
done!